opm/output/test_util/EclFilesComparator.hpp

```diff
@@ -0,0 +1,108 @@
+/*
+   Copyright 2016 Statoil ASA.
+
+   This file is part of the Open Porous Media project (OPM).
+
+   OPM is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   OPM is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+   */
+
+#ifndef ECLFILESCOMPARATOR_HPP
+#define ECLFILESCOMPARATOR_HPP 
+
+#include <vector>
+#include <string>
+
+/// Prototype for eclipse file struct, defined in the ERT library
+struct ecl_file_struct;
+typedef struct ecl_file_struct ecl_file_type;
+/// Prototype for eclipse grid struct, defined in the ERT library
+struct ecl_grid_struct;
+typedef struct ecl_grid_struct ecl_grid_type;
+
+enum eclFileEnum {RESTARTFILE, INITFILE};
```

Comment:
Yeah, I thought about that. The reason why I figured I'd create my own was that I only want restart, init, rft(?) .. and not summary files since they are covered by #56, and I would also have to include the entire ecl_util.h only to be able to use the small enum def – which seemed unnecessary.